### PR TITLE
ticdc: Fix some ticdc docs master 6.5

### PR DESCRIPTION
**Pull request by asddongmen**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-changefeed-config.md [hunk 1]
```diff
@@ -70,24 +70,24 @@ enable-old-value = true
...
 # 第二个事件过滤器规则
-[[filter.event-filters]]
-matcher = ["test.fruit"] # 该事件过滤器只应用于 test.fruit 表
-ignore-event = ["drop table"] # 忽略 drop table 事件
-ignore-sql = ["delete"] # 忽略 delete DML
-ignore-insert-value-expr = "price > 1000 and origin = 'no where'" # 忽略包含 price > 1000 和 origin = 'no where' 条件的 insert DML
+# [[filter.event-filters]]
+# matcher = ["test.fruit"] # 该事件过滤器只应用于 test.fruit 表
+# ignore-event = ["drop table", "delete"] # 忽略 drop table 的 DDL 事件和 delete 类型的 DML 事件
+# ignore-sql = ["^drop table", "alter table"] # 忽略以 drop table 开头的，或者包含 alter table 的 ddl 语句
```
Comment (ran-huang) [inline ticdc/ticdc-changefeed-config.md:89]:
```suggestion
# ignore-sql = ["^drop table", "alter table"] # 忽略以 drop table 开头的，或者包含 alter table 的 DDL 语句
```

File: ticdc/ticdc-changefeed-config.md [hunk 2]
```diff
@@ -101,34 +101,34 @@ region-threshold = 100000
...
+# protocol = "canal-json"
 
 # 以下三个配置项仅在同步到存储服务的 sink 中使用，在 MQ 和 MySQL 类 sink 中无需设置。
 # 换行符，用来分隔两个数据变更事件。默认值为空，表示使用 "\r\n" 作为换行符。
-terminator = ''
+# terminator = ''
 # 文件路径的日期分隔类型。可选类型有 `none`、`year`、`month` 和 `day`。默认值为 `none`，即不使用日期分隔。详见 <https://docs.pingcap.com/zh/tidb/dev/ticdc-sink-to-cloud-storage#数据变更记录>。
 date-separator = 'none'
-# 是否使用 partition 作为分隔字符串。默认值为 true，即一张表中各个 partition 的数据会分不同的目录来存储。详见 <https://docs.pingcap.com/zh/tidb/dev/ticdc-sink-to-cloud-storage#数据变更记录>。
+# 是否使用 partition 作为分隔字符串。默认值为 true，即一张表中各个 partition 的数据不会分不同的目录来存储。详见 <https://docs.pingcap.com/zh/tidb/dev/ticdc-sink-to-cloud-storage#数据变更记录>。
```
Comment (ran-huang) [inline ticdc/ticdc-changefeed-config.md:121]: "不会分不同的目录" 的意思是 “会使用相同的目录”吗？双重否定不太好理解

Comment (CharlesCheung96) [inline ticdc/ticdc-changefeed-config.md:121]:
```suggestion
# 是否使用 partition 作为分隔字符串。默认值为 true，即一张表中各个 partition 的数据会分不同的目录来存储。详见 <https://docs.pingcap.com/zh/tidb/dev/ticdc-sink-to-cloud-storage#数据变更记录>。
```

Comment (CharlesCheung96) [reply ticdc/ticdc-changefeed-config.md:121]: 这里写错了

**Review by ran-huang — APPROVED** [review]

**Comment by asddongmen** [comment]
@CharlesCheung96  PTAL, thank you~

**Review by CharlesCheung96 — APPROVED** [review]

**Pull request merged**